inc/extras.php

```diff
@@ -42,6 +42,16 @@ function alcatraz_body_classes( $classes ) {
 		$classes[] = 'header-style-' . esc_attr( $options['header_style'] );
 	}
 
+	// Mobile navigation toggle style class.
+	if ( isset ( $options['mobile_nav_toggle_style'] ) && $options['mobile_nav_toggle_style'] ) {
```

Comment:
Your `( isset (` looks lonely all by itself. Try `( isset(`. It's a function you're calling. () after a thing calls the thing. That's also why in the navigation.js file we structure it like `( function() {} )()`, or more simply, `( thing )()`, where inside the first parenthesis we put a thing, and then second parenthesis after it immediately calls it. Inside the second parenthesis we can also put args, so in navigation.js it's `( function( $ ) { /* Some stuff */ } )( jQuery )`, where we take this object called jQuery and pass it into our anonymous function as the dollar sign, so then we can refer to it as $ inside our function.